### DISCUSSION TITLE
Web rebase results in unverified commits

### DISCUSSION THREAD

**Discussion by pzcfg** [community/community#12613] [Pull Requests]
My team was very excited to see the ability to rebase PRs on the web, however we're unable to use it because it does not play well with verified commits.

When we rebase other people's commits, it results in an Unverified state, which we don't allow to be merged.

<img width="868" alt="Unverified" src="https://user-images.githubusercontent.com/94077014/157522945-e5e702c6-9519-40b0-95b2-46d5e5e05449.png">

When I do `git rebase <branch>` from my command line and force push, it results in `Partially verified` commits, which are allowed to be merged.

<img width="862" alt="Partially verified" src="https://user-images.githubusercontent.com/94077014/157523070-96a6958d-1d93-4b25-99f4-9c65deb1cf71.png">

I could understand this not being possible from the web, except that I noticed when I use the "Edit file" feature on the web, the resulting commit is Verified, so it seems that same functionality should also be able to verify the rebased commits? Please let me know if this is possible as we really would love to be able to use this feature.

**Comment by caugner** [comment]
Duplicate of #11639.

**Reply by ramayer** [reply]
Is there a bug report for this?

**Reply by adlai** [reply]
Honestly, this is due to how git works, rather than specific fault of GitHub. Although it is possible to argue verification for patches by reference to the signed objects, this would be difficult in some cases, hopeless in most, and misleading in worst.
